instructor-training-process.md

```diff
@@ -0,0 +1,154 @@
+# Trainer Training Process
+
+This document describes the workflow for training new instructors,
+and is meant as both a guide to trainers
+and as a checklist for adding support for this workflow to [AMY][amy].
+
+## Workflow
+
+1.  **Recruiting candidates.**
+    1.  People may apply directly as individuals to become instructors
+        by filling in a form on our website.
+    1.  People may apply in physically co-located groups.
+        We have only done this one (for training in Dec 2015),
+	but it worked so well that we would like it to become routine.
+	In this case,
+	groups need to fit between a minimum and maximum size (e.g., 3 to 12)
+	and the application must includes names and contact information for everyone who intends to take part.
+	Groups must also commit to organizing and running at least one workshop within a few months of training.
+    1.  We may commit to training people for partner organizations as part of their partnership agreements.
+        Here,
+	we negotiate dates with the partner site,
+	which selects the participants subject to the same per-site size restrictions above (3-12 people per site).
+1.  **Matching candidates to courses.**
+    This has been done manually to date:
+    the lead trainer, members of the Steering Committee, and/or the Executive Directors
+    decide on dates and then invite individuals and/or groups to take part.
+1.  **Running courses.**
+    1.  In-person courses are run over two days,
+        with all participants at one site.
+    1.  Live online courses are also run over two days,
+        but participants take part in small groups at a small number of sites
+	(typically 3-12 per site and 2-4 sites).
+    1.  Asynchronous online courses are run over several weeks.
+        Participants are typically *not* co-located,
+	but get together for an hour of discussion online each week.
+	This format is being phased out,
+	as it took more effort from trainers
+	and had poorer follow-through rates than the intensive two-day versions.
+1.  **Homework.**
+    1.  After completing the course,
+        the trainee picks one core lesson from each carpentry she wants to teach
+        and submits a small change for that lesson.
+    1.  Changes do *not* have to be accepted in order for the trainee to move on,
+        but they do have to be properly formatted.
+    1.  Changes are submitted as GitHub pull requests for Software Carpentry
+        and by appending them to a Google Docs page for Data Carpentry
+	(because Data Carpentry doesn't teach Git,
+	and therefore doesn't want to require instructors to have to know it).
+    1.  Changes to content are discouraged,
+	since the lessons are now fairly stable.
+    1.  New exercises are also becoming problematic,
+	since we already have more than we can possibly use.
+    1.  Load on lesson maintainers has become unsustainable.
```

Comment:
Why is the load on SWC lessons maintainers unsustainable? In shell/git/python SWC lessons repositories, I can see only 40/20/20 new pull requests and 8/8/0 new issues created since February 2016. Most of the pull requests are small and look like they could be merged without changes.